### PR TITLE
Removed redundant code and updated SDK

### PR DESCRIPTION
**Pull request by mtrojanowski**

### CODE REVIEW

File: .github/workflows/maven.yml [hunk 1]
```diff
@@ -1,25 +0,0 @@
-# This workflow will build a Java project with Maven, and cache/restore any dependencies to improve the workflow execution time
```
Comment (mtrojanowski) [inline .github/workflows/maven.yml:1]: I removed this file completely as Github would require access to Secmakrer's wsdl to be able to build the project.

**Review by ju-cu — APPROVED** [review]

File: .github/workflows/maven.yml (hunk 1 above)
Comment (renatoathaydes) [reply .github/workflows/maven.yml:1]: That's a shame, but yeah, looks impossible to solve that.

**Review by renatoathaydes — APPROVED** [review]

File: .github/workflows/maven.yml (hunk 1 above)
Comment (daniellindau) [reply .github/workflows/maven.yml:1]: Can we do it now that the wsdl's are committed to the repo?

Comment (mtrojanowski) [reply .github/workflows/maven.yml:1]: Definitely. I will bring back the workflow file.

**Comment by mtrojanowski** [comment]
The relevant changes from this PR are incorporated in #9.

**Pull request closed without merging**